daemon/logger/jsonfile/jsonfile.go

```diff
@@ -1,32 +1,68 @@
 package jsonfile
 
 import (
+	"fmt"
 	"os"
+	"strconv"
 	"sync"
 
 	"github.com/alibaba/pouch/daemon/logger"
+	"github.com/alibaba/pouch/pkg/bytefmt"
 )
 
 // JSONLogFile is uses to log the container's stdout and stderr.
 type JSONLogFile struct {
-	mu sync.Mutex
-
-	f      *os.File
-	perms  os.FileMode
-	closed bool
+	mu          sync.Mutex
+	f           *os.File
+	perms       os.FileMode
+	closed      bool
+	maxSize     uint64 // maximum size of log in byte
+	currentSize uint64 // current size of the latest log in byte
+	maxFile     int    // maximum number of logs
 }
 
 // NewJSONLogFile returns new JSONLogFile instance.
-func NewJSONLogFile(logPath string, perms os.FileMode) (*JSONLogFile, error) {
+func NewJSONLogFile(logPath string, perms os.FileMode, logConfig map[string]string) (*JSONLogFile, error) {
```

Comment:
I prefer to define the `logConfig` to a struct but just a `map[string]string`, WDYT? @zjumoon01